docs/content/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -153,8 +153,50 @@ state goes from edge to the mailbox workspace and then is summarized
 to the workload management workspace.  State propagation is maintained
 in an eventually consistent way, it is not just one-and-done.
 
+## Note on milieu
+
+We have made a terminoloogy shift since this document was originally
+written, but not yet thoroughly updated this document to use the new
+terminology. Both old and new terminology and their relation is
+documented in [the
+glossary](../../../Getting-Started/user-guide/#glossary).
+
+This design is set in the context of kcp and currently in the midst of
+a transition away from that. This design uses kcp workspaces as
+spaces, and eschews the use of kcp APIExport+APIBinding in favor of
+using [kube-bind](https://github.com/kube-bind/kube-bind).
+
+## Space identifiers
+
+The kube-bind module has its own identifiers for spaces. These are
+distinct from the ways that kcp identifies spaces. In kcp there are
+two ways of identifying most spaces: `logicalcluster.Name` and the
+pathname of the corresponding `Workspace` object. In this design the
+controllers have been using `logicalcluster.Name` as the space
+identifier (using the `Path` method of `logicalcluster.Name` when
+passing a `Name` to a func that takes a `Path`). There is a 1:1
+relation between the kube-bind ID for a space and its
+`logicalcluster.Name`. This relation is queried through the following
+public funcs.
+
+```go
+
+// SpaceIDToKubeBind maps a `logicalcluster.Name` to the ID
+// that kube-bind uses for that space. This is kube-bind's
+// "cluster namespace" in the provider's cluster.
+// Returns empty string if there is no relationship.
+func SpaceIDToKubeBind(spaceID string) string {..}
```

Comment:
This is not 1:1 in case a consumer binds to two providers. In KS we only have ESPW  so it may be OK.

Reply:
As you see, this has been addressed in later revisions.